Formula/libuninameslist.rb

```diff
@@ -5,6 +5,12 @@ class Libuninameslist < Formula
   sha256 "5c0283b2e18d101e58b70a026119d66c9d3e749e4537def7799bba0bc8998f62"
   license "BSD-3-Clause"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
+    regex(%r{href=.*?/tag/v?(\d+)["' >]}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)*)["' >]}i)
    strategy :github_latest
```

A couple things:

* `strategy` should come last in the `livecheck` block (after `regex` here). The idea is that if we were to add a `strategy` block in the future, we would need `regex` to be defined before the `strategy` block to be able to pass the `regex` in. Eventually I'll get around to adding a Rubocop for this.
* The `libuninameslist` repository contains older tags with a format like `0.5.20150701`. This updates the regex to use the `*` version of the standard regex, so it can optionally match versions with more than one part.

Reply:
> `strategy` should come last in the `livecheck` block

That is a good note for me.